Makefile

```diff
@@ -0,0 +1,38 @@
+SHELL := /bin/bash
+DEST := result
+PORT := 5000
+
+.PHONY: all clean html web compile serve
+
+UNAME = $(shell uname)
+ifeq ($(UNAME), Linux)
+CP_ARGS = -l
+endif
+ifeq ($(UNAME), Darwin)
+CP_ARGS =
+endif
+
+all: serve
+
+prep:
+	mkdir -p $(DEST)/nightly
+
+clean:
+	$(MAKE) -C guides/ clean
+	rm -rf $(DEST) web/output/
+
+html: prep
+	$(MAKE) -C guides/ html BUILD=foreman-el
+	$(MAKE) -C guides/ html BUILD=foreman-deb
+	$(MAKE) -C guides/ html BUILD=katello
+
+web: prep
+	pushd web; bundle exec nanoc; popd
+
+compile: web html
+	cp $(CP_ARGS) -nr web/output/* $(DEST)
+	cp $(CP_ARGS) -nr guides/build/* $(DEST)/nightly/
```

Comment:
Couldn't this instead be a symlink?

Reply:
Actually no, I tried and I had issues with overwriting existing symlinks. If you figure out (and test) a solution, let me know.